### DISCUSSION TITLE
offline audio to text

### DISCUSSION THREAD

**Discussion by ebrahim-zarei** [SubtitleEdit/subtitleedit#7663] [General]
offline audio to text is possible by subtitle edit?
Ebrahim.zarei@gmail.com

**Comment by Purfview** [comment]
Yes.

**Comment by niksedk** [comment]
You will have to be online the first time, where you will need to download Whisper engine + models. After this, no online connection is needed.
Read more here: https://www.nikse.dk/subtitleedit/help#audio_to_text

**Reply by ebrahim-zarei** [reply]
thanks a lot
